apps/protocol-frontend/src/components/ProfileForm.tsx

```diff
@@ -152,7 +186,8 @@ const ProfileForm = () => {
               <Heading as="h4" size="sm" fontWeight="medium" color="gray.700">
                 Linear
               </Heading>
-              {userData?.linear_users && userData.linear_users.length > 0 ? (
+              {userData?.linear_users?.length &&
+              userData.linear_users[0].active_token ? (
```

Comment:
Ah! So when a user disconnects do we then remove the `active_token`?

Reply:
`active_token` is set to false upon disconnecting;